test/rate-limit-policies.yml

```diff
@@ -25,3 +25,6 @@ registrationsPerIP:
 pendingAuthorizationsPerAccount:
   window: 168h # 1 week, should match pending authorization lifetime.
   threshold: 3
+certificatesPerNameSet:
```

Comment:
Let's call this certificatesPerFQDNSet to avoid confusion with certificatesPerName (where Name means something different).